v1 recipe, noarch: python

bollwyvl opened the pull request:
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

References:
- alternative to (and includes) #36

Changes:
- [x] `conda-forge.yml`
  - [x] use `rattler-build`
  - [x] use `pixi`
  - [x] use `hint-grayskull` for more detailed PR future summaries (hopefully)
  - [x] use `noarch_platforms` to continue testing on `osx_64` and `win_64`
    - > these will try to generate the same build hash, e.g. `pyhcf101f3_2`, so practically only the `linux_64` build will be uploaded as it will generally finish first
- [x] `recipe/recipe.yaml`
  - [x] use `noarch: python` to avoid #36 (and most future python/arch-specific migrations)
  - [x] sync with upstream `pyproject.toml`
    - [x] update pyparsing dependency 
    - [x] update URLs
  - [x] run test suite with coverage
  - [x] pip check/import test on minimum build python and a fixed "non-build" python
  - [x] add maintainer :wave:

regro-cf-autotick-bot committed:
Rebuild for python 3.14 freethreading

regro-cf-autotick-bot committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.08.07.28.3

Commit:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.19.17.19.3

bollwyvl committed:
v1 recipe, noarch

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.19.20.45.22

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
fix tests

bollwyvl committed:
add maintainer

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.20.03.23.59

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.5

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.20.03.23.59

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.5

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

bollwyvl commented:
@conda-forge/pydot ready for review

nehaljwani commented:
Making this noarch will run the tests only on Linux. Is it possible to test on all platforms using the CI, but still ship only a noarch package?

I'm okay with merging this change unless @leycec says otherwise.

bollwyvl commented:
"conda-forge is not upstream CI," but yes, it is possible to test on multiple platforms with `noarch_platforms`. It's highly likely only the linux ones will actually upload, as they will likely finish first.

bollwyvl committed:
add some more noarch platforms

bollwyvl committed:
MNT: Re-rendered with conda-smithy 3.54.1 and conda-forge-pinning 2025.12.20.18.11.09

Other tools:
- conda-build 25.11.1
- rattler-build 0.54.0
- rattler-build-conda-compat 1.4.5

bollwyvl commented:
Thanks!
